lib/bosh/cli/commands/prepare.rb

```diff
@@ -26,6 +26,7 @@ def prepare
 
     def prepare_release_repos
       project_deployment_releases.each do |release|
+        next if release.git_url.empty?
```

Comment:
Could you add a spec for this behaviour in [prepare_spec](https://github.com/wayneeseguin/bosh-workspace/blob/gitoptional/spec/commands/prepare_spec.rb) so that further version will not break the optional git url support.